tests/functional/resources/modules/d8/woot/woot.deploy.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Successful deploy hook.
```

Comment:
Since this is our only docs of a new hook, lets add doxygen in all functions - `Implements HOOK_deploy_NAME`

Reply:
hmm but as for post_update hooks the comment is used in the display of the pending update functions. I don't mind either way..